src/service/smarthome/api/_test/v1.test.ts

```diff
@@ -20,3 +20,10 @@ import * as Api from '../v1'
 test('api is an object', t => {
   t.is(typeof Api, 'object')
 })
+
+test('only command field is mandatory in SmartHomeV1ExecuteRequestExecution', t => {
+  const executeRequest: Api.SmartHomeV1ExecuteRequestExecution = {
```

Comment:
Not to be too strict, but can you just make this variable a copy of the [Dock execute request](https://developers.google.com/actions/smarthome/traits/dock#device-commands) so that the test has a practical basis?

Reply:
Done